### PR TITLE
comedy movies

### PR DESCRIPTION
**Pull request by RajivSah**
add comedy movies list

### CODE REVIEW

**Review by shankar524 — COMMENTED** [review]
Add more items

File: comedy.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Movie list
```
Comment (subeshb1) [inline comedy.md]: Capitalize 'L' in list.

Comment (RajivSah) [reply comedy.md]: @subeshb1 done changes

Comment (shankar524) [reply comedy.md]: nice work @Rxyz58

**Pull request closed without merging**